test.js

```diff
@@ -15,7 +23,7 @@ var testCommon = suite.common({
   // Unsupported features
   createIfMissing: false,
   errorIfExists: false,
-  seek: false
+  seek: true
```

Comment:
I think this line can be removed now if the default is true?

Reply:
It also serves as a signal of what this implementation supports, let's keep it

Reply:
(A signal to humans, I mean)